content/06-Combining-Subschemas/04-Valid-Against-oneOf-the-Subschemas(XOR)/instructions.mdx

```diff
@@ -52,7 +52,7 @@ We will combine using `oneOf` and `required`, to ensure that only one of the pro
 }
 ```
 
-> Notice the `format` keyword with `date` used in the `dateOfBirth` property. By default, `format` is just an annotation and **does not effect validation**. you can learn more about the format keyword [here](https://json-schema.org/understanding-json-schema/reference/string#format).
```

Comment:
Hi @srivastava-diya, the link should point to the original json-schema.org website, not our tour. The link is a reference to the documentation link. please look up the website and find the relevant section and use the link here.